docs/tutorial/cspc/allow-tagged-bds/allowed-bds.md

```diff
@@ -0,0 +1,144 @@
+
+```bash
+$ kubectl apply -f cspc.yaml 
+cstorpoolcluster.cstor.openebs.io/cspc-stripe created
+$ kubectl get cspi -n openebs
+NAME               HOSTNAME           FREE     CAPACITY    READONLY   PROVISIONEDREPLICAS   HEALTHYREPLICAS   STATUS   AGE
+cspc-stripe-b9f6   worker2   19300M   19300614k   false      0                     0                 ONLINE   89s
+cspc-stripe-q7xn   worker1   19300M   19300614k   false      0                     0                 ONLINE   89s
+```
+
+You can see that CSPI for node `worker3` is not created and the reason for this
+is the following:
+- CSPC YAML created above has `openebs.io/allowed-bd-tags: cstor,ssd` in its 
+annotation. This means that the CSPC operator will only consider those block
+devices for provisioning that do not have a BD tag `openebs.io/block-device-tag` 
+on the block device or has the tag but the values are either `cstor` or `ssd`.
+
+- In this case the `blockdevice-022674b5f97f06195fe962a7a61fcb64` 
+(on node `worker1`) and `blockdevice-241fb162b8d0eafc640ed89588a832df`
+(on node `worker2`) does not have a tag with key `openebs.io/block-device-tag`.
+Hence, not restriction is applied on it and it can be used the the CSPC operator
+for pool provisioning or operations.
+
+- In the case of blockdevice `blockdevice-00439dc464b785256242113bf0ef64b9`
+(on node `worker3`) it has a block device tag `openebs.io/block-device-tag` with
+value `fast`. (This tag is simply Kubernetes label). But on the CSPC, the
+annotation `openebs.io/allowed-bd-tags` has value `cstor` and `ssd`. There is no
+`fast` keyword present in the annotation value and hence this BD cannot be used. 
+
+Now edit this CSPC to add `fast` in the annotation `openebs.io/allowed-bd-tags`
+via Kubectl. 
+
+```yml
+openebs.io/allowed-bd-tags: cstor,ssd,fast
+
+```
+After adding the annotation on the CSPC the CSPI will get created.
+
+```bash
+$ kubectl get cspi -n openebs
+NAME               HOSTNAME           FREE     CAPACITY    READONLY   PROVISIONEDREPLICAS   HEALTHYREPLICAS   STATUS   AGE
+cspc-stripe-b9f6   worker2   19300M   19300074k   false      0                     0                 ONLINE   7m8s
+cspc-stripe-lznh   worker3   19300M   19300053k   false      0                     0                 ONLINE   5s
+cspc-stripe-q7xn   worker1   19300M   19300074k   false      0                     0                 ONLINE   7m8s
+```
+
+## More Info
+- If you want BD of multiple tag values to be allowed, the value for allowed bd 
+tag annotation can be written in the following comma-separated manner:
+```yml
+openebs.io/allowed-bd-tags: fast,ssd,nvme
+```
+- A BD tag has only one value on the block device CR. For example 
+    - `openebs.io/block-device-tag: fast` — fast is the value.
+    - `openebs.io/block-device-tag: fast,slow` — one can tag a BD like this but 
+    it is not recommended and will not be allowed to be consumed by cStor as 
+    cStor annotation takes a comma-separated value for allowed bd tags.
```

Comment:
This sentence is not clear.

Reply:
Suggestions... I am not able to bring any other statement.

Reply:
> `openebs.io/block-device-tag: fast,slow` — one can tag a BD like this

this should never be done. should change the not recommended word.

The cstor annotation and block-device-tag label is getting mixed up here. Not able to understand it clearly.

If possible split into 2 statements 

Also the `fast-slow` brings in additional complexity in understanding.